lib/clearance/configuration.rb

```diff
@@ -18,6 +19,7 @@ def initialize
       @allow_sign_up = true
       @cookie_expiration = ->(cookies) { 1.year.from_now.utc }
       @cookie_path = '/'
+      @cookie_name = 'remember_token'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.